arch/arm64/boot/dts/qcom/msm8996.dtsi

```diff
@@ -1777,6 +1778,7 @@
 			snps,nominal-elastic-buffer;
 			snps,is-utmi-l1-suspend;
 			snps,hird-threshold = /bits/ 8 <0x0>;
+			qcom,usb-core-id = <0>;
```

Comment:
7b30fb617e47ee6b7efc2af659fffcde7f2789cf

Reply:
Dropped since we don't use it.